reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt

```diff
@@ -113,9 +113,11 @@ class ReportTableModelMapper(private val resolutionProvider: ResolutionProvider)
                     it.summary.errors
                 }?.distinct() ?: emptyList()
 
+                val packageForId = ortResult.getPackage(id)?.pkg ?: ortResult.getProject(id)?.toPackage()
```

Comment:
commit-msg: When reading this commit message the first questions popping up were
- `what else` is affected besides perma links
- is the change ok for other affected things

My understanding is that `nothing else` (beside perma links) is affected by this change.
Would you mind changing the commit message to make this more clear?

Reply:
I hope to have made this clearer now.

Reply:
Thanks!